### ISSUE TITLE
Compiling error re Java SE 9

### ISSUE THREAD

**Issue by bdunkleysmith** [WalkerKnapp/devolay#5]
Thanks for publishing this NDI project. I am excited about the opportunities it will open up for me.

I'm a little out of my comfort (competency) zone here, but I have encountered a couple of compiling "roadblocks".

Referring to 
[Initial error.txt](https://github.com/WalkerKnapp/devolay/files/3937493/Initial.error.txt) firstly I encountered:

> Task :devolay-java:generateJniHeaders FAILED
javac: directory not found: build/tmp/generateJniHeaders

I overcame that error by manually creating build/tmp/generateJniHeaders and then referring to 
[Console.txt](https://github.com/WalkerKnapp/devolay/files/3937503/Console.txt) I encountered:

> Task :examples:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':examples:compileJava'.
> Could not target platform: 'Java SE 9' using tool chain: 'JDK 8 (1.8)'.

I understood from comments in [Issue #3](https://github.com/WalkerKnapp/devolay/issues/3) that your wrapper was now Java 8 compatible and so what do you think may be causing my compliation problem?

Thanks in anticipation of your support.

**Comment by WalkerKnapp** [comment]
Thank you for opening an issue, it's great to see people making use of this project.

The `examples` subproject is built along with the normal assembly, but still relies on Java 9. Can you rerun the build with `./gradlew devolay-java:assemble`? This should exclude the examples.

Also, are you compiling yourself for any particular reason? It may be easier to manage an end product using the precompiled bintray builds if you aren't familiar with the Gradle native build system.

Again, thanks for reaching out, and I hope this helps!

**Comment by bdunkleysmith** [comment]
Your comments made me realise that I didn't have to compile myself, but could use your already precompiled binary. 

Coding is not my profession and any formal training I did was in Basic (and Fortran) and so I'm not comfortable in the Java, C, etc. environment. That's why I have adopted use of [B4J (basic for Java)](https://www.b4x.com/b4j.html) and so I am trying to use your project as the basis for a B4J library.

I have achieved a working library and created a simple testbed project which can find/identify NDI sources and send some frames (based on your SendVideoExample). However for some reason the way I have created the associated .xml file required for a B4J library, the methods are not being exposed correctly. 

I'm happy for you to close this issue, but I may contact you once I've finally completed the B4J library just so you are aware how your project can be used on another application development platform.
